File: src/B_10768/B_10768.java
package B_10768;

import java.io.*;

public class B_10768 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int m = Integer.parseInt(br.readLine());
        int d = Integer.parseInt(br.readLine());
        if(m==2){
            if(d==18){
                System.out.println("Special");
            }else{
                System.out.println(BA(d>18));
            }
        }else{
            System.out.println(BA(m>2));
        }

    }
    static String BA(boolean i){
        if (i) {
            return "After";
        } else {
            return "Before";
        }
    }
}

File: src/B_10768/B_10768.kt
package B_10768

import java.io.*

fun main() = with(BufferedReader(InputStreamReader(System.`in`))){
    val m: Int = readLine().toInt()
    val d: Int = readLine().toInt()
    if (m == 2) {
        if (d == 18) {
            println("Special")
        } else {
            println(BA(d > 18))
        }
    } else {
        println(BA(m > 2))
    }
}
fun BA(i: Boolean): String {
    return if (i) {
        "After"
    } else {
        "Before"
    }
}